content/en/monitors/monitor_types/process.md

```diff
@@ -11,74 +11,48 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
 ## Overview
 
-Process monitors are configured to check the status of individual or groups of processes. Process monitors are evaluated every minute.
+Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the behavior of any group of processes across hosts or tags.
 
-## Live Process Monitors
+## Monitor creation
 
-{{< img src="monitors/monitor_types/process/live_process_monitor_select.png" alt="live process monitor" responsive="true" style="width:35%;">}}
+To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
 
-Live Process Monitors are based on data collected by the [Process Agent][1] which allows you to centrally create monitors that warn or alert based on the behavior of any group of processes across any hosts or tags.
+### Select processes
 
-### Configuration
+Search for a process to monitor using a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure. Use tags to refine your monitor to a specific scope.
 
-1. **Search for the Process to monitor.**
-  This can be a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure.
-  Matching processes and counts are displayed in the table below:
+Matching processes and counts are displayed below the search:
 
-    {{< img src="monitors/monitor_types/process/search_process.png" alt="Search process" responsive="true" style="width:80%;">}}
+{{< img src="monitors/monitor_types/process/select_processes.png" alt="Select processes" responsive="true" style="width:90%;">}}
 
-2. **Use tags to refine your monitor to a specific scope.**
-  You only see hosts or tags that are reporting a status for the selected process.
+After defining your search, a graph is displayed above the search inputs with the total number of processes found.
 
-    {{< img src="monitors/monitor_types/process/selecting_scope.png" alt="Selecting scope" responsive="true" style="width:80%;">}}
+#### Alert grouping
 
-    **Note**: Above the *Select process* section is displayed the graph of the number of processes in the scope for the monitor over time.
-    Selecting `multi-alert` split this graph into one line per group.
+`Simple Alert` (default): aggregates alerts over all reporting sources. You receive one alert when the aggregated value meets the set conditions.
```

Comment:
```suggestion
`Simple Alert` (default): aggregates alerts over all reporting sources. You receive one notification when the aggregated value meets the set conditions.
```

Reply:
I think alert is a better word here since people can set up monitors that don't send out notifications.